src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+export const MovieCard: React.FC<{ movie: Movie }> = ({ movie }) => (
```

Comment:
Use interface for passing props, it will clean up your code.
Also you can import FC directly from ''react"
```suggestion
interface Props {
  movie: Movie
}
export const MovieCard: FC<Props> = ({ movie }) => (
```